packages/airnode-deployer/src/cli/__tests__/commands.test.ts

```diff
@@ -22,8 +22,8 @@ import { Receipt } from '../../types';
 describe('deployer commands', () => {
   it('can deploy Airnode', async () => {
     await deploy(
-      join(__dirname, '../../../config/config.json.example'),
-      join(__dirname, '../../../config/secrets.env.example'),
+      join(__dirname, '../../../config/config.example.json'),
```

Comment:
I suppose it's better to have it this way for syntax highlighting?

Reply:
Yeah, this is tracked in https://api3dao.atlassian.net/browse/AN-424 and I thought I will do this to simplify the rename from 0.5 to 0.4 (I thought replacing in *.json files would be enough). 

It turned out that it isn't enough though.